tools/cockpit.spec

```diff
@@ -347,8 +347,8 @@ Cockpit support for reading PCP metrics and loading PCP archives.
 %package dashboard
 Summary: Cockpit SSH remoting and dashboard
 Requires: libssh >= %{libssh_version}
-Requires: cockpit-ws = %{version}-%{release}
-Provides: cockpit-ssh = %{version}-%{release}
+Requires: cockpit-bridge > 134
```

Comment:
Unfortunately ```required_base``` doesn't yet work for the ```cockpit-system``` package ... which really is a bundle of other packages. I'll leave this manually set in the spec file for now. Do you have ideas on how to make ```gen-spec-dependencies``` work for ```cockpit-system```?